Tests/UIKitTests/UINavigationControllerExtensionsTests.swift

```diff
@@ -53,5 +53,19 @@ final class UINavigationControllerExtensionsTests: XCTestCase {
         XCTAssertNotNil(color)
         XCTAssertEqual(color!, .red)
     }
+    
+    func testHideBottomBar() {
+        let rootVC = UIViewController()
+        let navigationController = UINavigationController(rootViewController: rootVC)
+        let vcToPush = UIViewController()
+        let tabVC = UITabBarController()
+        tabVC.viewControllers = [navigationController]
+        navigationController.pushViewController(vcToPush, hideBottomBar: true, animated: true)
```

Comment:
Don't animate in unit tests.
```suggestion
        navigationController.pushViewController(vcToPush, hideBottomBar: true, animated: false)
```